### PR TITLE
[restinio] Update to 0.7.0

### PR DESCRIPTION
**Pull request by eao197**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- ~~[ ] The "supports" clause reflects platforms that may be fixed by this new version~~
- ~~[ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- ~~[ ] Any patches that are no longer applied are deleted from the port's directory.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by Cheney-W** [comment]
Usage passed with following triplets:
x64-windows 
x64-windows-static

File: ports/restinio/portfile.cmake [hunk 1]
```diff
@@ -2,18 +2,29 @@ vcpkg_from_github(
...
+        -DRESTINIO_DEP_EXPECTED_LITE=find
 )
 vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/restinio)
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")
 
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
+
+# vim:expandtab:ts=4:sw=4:sts:
+
```
Comment (Cheney-W) [inline ports/restinio/portfile.cmake:30]: Are these comments useful? If it doesn't work, please delete them.

Comment (eao197) [reply ports/restinio/portfile.cmake]: Done.

Comment (Cheney-W) [reply ports/restinio/portfile.cmake]: Thank you for your prompt response.

**Comment by Cheney-W** [comment]
Please click the `Read for review` button once you finished this PR.

**Review by Cheney-W — APPROVED** [review]

**Comment by faqeerhussain106** [comment]
> * [x]  Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
> * [x]  SHA512s are updated for each updated download
> * ~[ ] The "supports" clause reflects platforms that may be fixed by this new version~
> * ~[ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~
> * ~[ ] Any patches that are no longer applied are deleted from the port's directory.~
> * [x]  The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
> * [x]  Only one version is added to each modified port's versions file.

**Review by data-queue — APPROVED** [review]

**Pull request merged**